File: cakk-api/src/main/java/com/cakk/api/Application.java
package com.cakk.api;

import java.util.TimeZone;

import jakarta.annotation.PostConstruct;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class Application {

	@PostConstruct
	public static void started() {
		TimeZone.setDefault(TimeZone.getTimeZone("Asia/Seoul"));
	}

	public static void main(String[] args) {
		started();
		SpringApplication.run(Application.class, args);
	}
}

File: cakk-api/src/main/kotlin/com/cakk/api/Application.kt
package com.cakk.api

import java.util.*

import jakarta.annotation.PostConstruct

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class Application

@PostConstruct
fun started() {
	TimeZone.setDefault(TimeZone.getTimeZone("Asia/Seoul"))
}

fun main(args: Array<String>) {
	started()
	runApplication<Application>(*args)
}

